### ISSUE TITLE
`mdbook` without arguments should show something useful

### ISSUE THREAD

**Issue by Havvy** [rust-lang-nursery/mdBook#617]
Here's what it currently shows:

```
$ mdbook
error: 'mdbook' requires a subcommand, but one was not provided

USAGE:
    mdbook <SUBCOMMAND>

For more information try --help
```

It should either just default to showing the help message after the error or list what the actual subcommands are.

**Comment by Michael-F-Bryan** [comment]
I believe this is the default behaviour for `clap`. There is a [setting] to display the help text if no arguments (or subcommands) are provided though. It should be trivial to update our `msbook` binary to enable that setting.

[setting]: https://docs.rs/clap/2.29.4/clap/enum.AppSettings.html#variant.ArgRequiredElseHelp

**Comment by Michael-F-Bryan** [comment]
Fixed in #624.
